### DISCUSSION TITLE
视频页新出现的“大家围观的直播”

### DISCUSSION THREAD

**Discussion by df4d** [the1812/Bilibili-Evolved#4003] [功能建议 / Ideas]
视频页多了“大家围观的直播”，和视频内容没有关联，还跟着滚动条一起滚动，希望大大们给个选项隐藏掉（大概就几行行css的工作量？）
![1676644135634](https://user-images.githubusercontent.com/88201634/219683489-a1d702cf-787e-4046-957b-3800913298b1.png)

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：#3993

**Reply by df4d** [reply]
我搜索的关键词问题，没搜到，我的锅
